c7n/actions.py

```diff
@@ -559,7 +564,7 @@ class AutoTagUser(EventAction):
 
     References
      - AWS Config (see REQUIRED_TAGS caveat) - http://goo.gl/oDUXPY
```

Comment:
@kapilt - the link provided does not have a `REQUIRED_TAGS` field.  Should we be linking to this instead?

http://docs.aws.amazon.com/config/latest/developerguide/required-tags.html